### ISSUE TITLE
Manage Workflow Documents action specifies DCTM-specific condition

### ISSUE THREAD

**Issue by nwoodbine** [tsgrp/hpi#737]
See manageworkflowdocuments.js line 82, objectversionlabelcontainslatest. This condition evaluator is not implemented for Alfresco or HBase.

**Comment by dgrumieaux** [comment]
This does not need to be implemented in Alfresco since the Wizard's versioning schema currently tells what is the latest document by simply tracking the only version of the document that is in the SpacesStore (as opposed to the versionStore)

**Comment by zkeller** [comment]
To clarify, though, the javascript still needs to be reworked to not attempt to evaluate conditions that aren't available/implemented for all repositories, since HPI is repository agnostic.

**Comment by dgrumieaux** [comment]
The conditions to be evaluated are configurable. Perhaps the issue here is to just have the configurations more easily discoverable for dctm and Alfresco.

**Comment by zkeller** [comment]
While configurable, I think the default conditions shouldn't reference conditions that won't work in all envs.

**Comment by dgrumieaux** [comment]
Yep, makes sense to me. Can we leave comments for suggested base conditions for each implementation, and force the project to configure that field? Or have a sane default that works for all potential implementations?

**Comment by dgrumieaux** [comment]
note this issue is somewhat related to this: https://github.com/tsgrp/OpenContent/issues/47

**Comment by dgrumieaux** [comment]
Fixing https://github.com/tsgrp/OpenContent/issues/47 may be a better fix to this issue - creating a generic condition that works on both Documentum and Alfresco. In the meantime, this issue can be worked around by having your individual project override the conditions as needed.
